core/jobs.py

```diff
@@ -123,7 +123,7 @@ def _create_new_job():
             job_id = job_models.JobModel.get_new_id(cls.__name__)
             job_models.JobModel(id=job_id, job_type=cls.__name__).put()
             return job_id
-
+        """Uses _create_new_job to create a new task having a job id"""
```

Comment:
Add docstrings only at the beginning of the method. You can refer other PRs as a reference. Please remove them.